packages/javascript/bh-shared-ui/src/components/ContentPage/ContentPage.tsx

```diff
@@ -32,9 +32,6 @@ const ContentPage: React.FC<ContentPageProps> = ({ title, children, actionButton
                         <Typography variant='h1'>{title}</Typography>
                         {actionButton}
                     </Box>
-                    <Box mt={2} mb={4}>
-                        <Divider />
-                    </Box>
```

Comment:
Removing this divider from the ContentPage component and also removing the actionButton prop as suggested in a different comment will make it essentially the same as the PageWithTitle component. We should probably scrap one or the other.

Reply:
The reason for the change was to make them stylistically consistent. The biggest remaining difference between them appears to be content offsets - ContentPage justifies all the way left, while PageWithTitle leaves some whitespace. It makes ContentPage appear nicer for standalone pages (Download Collectors, My Profile), while PageWithTitle is better for pages under Admin.

I'm happy to collapse the two. Is there a preference for which should remain?

Reply:
I would keep PageWithTitle and modify it like so:

```tsx
import { Box, Container, Typography, ContainerProps } from '@mui/material';
import React from 'react';

type PageWithTitleProps = ContainerProps<
    'div',
    {
        title?: string;
        pageDescription?: JSX.Element;
        children?: React.ReactNode;
    }
>;

const PageWithTitle: React.FC<PageWithTitleProps> = ({ title, pageDescription, children, ...rest }) => {
    return (
        <Container maxWidth='xl' {...rest}>
            <Box component={'header'} my={2}>
                {title && <Typography variant='h1'>{title}</Typography>}
                {pageDescription}
            </Box>
            {children}
        </Container>
    );
};

export default PageWithTitle;
```